srv/modules/runners/iscsi_upgrade.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+"""
+NFS-Ganesha Upgrade script
```

Comment:
Seems that this text "NFS-Ganesha Upgrade script" needs to be changed to "iSCSI Upgrade script" as this is not the iSCSI upgrade script I understand ;-)

Reply:
good catch! fixed